src/server/graphql/mutations/inactivateUser.js

```diff
@@ -0,0 +1,79 @@
+import {GraphQLBoolean, GraphQLID, GraphQLNonNull} from 'graphql';
+import adjustUserCount from 'server/billing/helpers/adjustUserCount';
+import getRethink from 'server/database/rethinkDriver';
+import {requireOrgLeaderOfUser, requireWebsocket} from 'server/utils/authorization';
+import {toEpochSeconds} from 'server/utils/epochTime';
+import {MAX_MONTHLY_PAUSES, PAUSE_USER} from 'server/utils/serverConstants';
+import {errorObj} from 'server/utils/utils';
+
+export default {
+  type: GraphQLBoolean,
+  description: 'pauses the subscription for a single user',
```

Comment:
oh this is where things get spicy.
the `pause` status of a user can be triggered by an organization leader, but it affects that users across all of his orgs. For example, if you go on vacation, i might pause you. but if you log in, we'll automatically unpause you & show you all of your projects across all of your orgs. so, the "pause" concept is tied to a user, but since each org has a unique billing cycle, we pro-rate differently.

Reply:
Hmm. This seems like a sensitive use case I'd want to track as we get more qualitative feedback. I'm not stoked on the idea of Organization A being able to pause User U in organizations A, B, and C. Likewise, if a user happens to be active in Organization A, but inactive in B, I don't see why we'd want to be be billing B.

Reply:
oh yeah, we discussed this one in depth. there's no great solution, but i think we found the only _good_ solution. The discussion is in a few older docs, but here's the gist:

how can you be active in only 1 org? that means we'd have to hide _some_ projects from you. If we do that, then you have to click an "unpause" button. If it requires user interaction to unpause, that means someone is gonna not see the button & start screaming "WHERE DID MY CARDS GO". If an employee costs $100/hr and necessitating a click will realistically save the organization $2, but cost someone 30 mins of their time, we just cost them $48 and all the sudden they are NOT stoked on our app. The delightfulness of saving them money goes away real quick when it causes confusion.